common/changes/@kadena/pactjs-generator/fix-client-convert-decimal-integer-ipactdecimal-ipactinteger_2023-02-02-14-57.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@kadena/pactjs-generator",
+      "comment": "BREAKING: Pact.modules functions will accept for numbers only IPactDecimal or IPactInteger",
```

Comment:
Since it is a breaking change, shouldn't we mark the type as minor? or major?

Reply:
Technically yes, but we're still in alpha versions and if we set this to minor/major it'll update the version accordingly... minor disadvantage of rush